screenpy/actions/tryto.py

```diff
@@ -0,0 +1,79 @@
+"""
+try/accept logic using screenplay pattern
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from screenpy.pacing import the_narrator
+from screenpy.speech_tools import get_additive_description
+
+if TYPE_CHECKING:
+    from screenpy import Actor
+    from screenpy.protocols import Performable
+
+
+class TryTo:
+    """
+    TryTo perform followup action if the first one fails.
+
```

Comment:
I know this is very nitpicky, but the docstring style in the other Actions is to have a succinct first line right after the """s, followed by additional verbiage explaining the Action. I would like to see some mention of try/except blocks in the followup section, so maybe like this:

```python
class TryTo:
    """Perform an Action, and another one if that fails.
    
    Allows Actors to mimic a try/except block, but all within an 
    :meth:`~screenpy.actor.Actor.attempts_to` call.
    ...
    """
```

something like that!

Reply:
Take a look at the latest update.